### DISCUSSION TITLE
Project site shows organization site

### DISCUSSION THREAD

**Discussion by magnetenstad** [community/community#21622] [Other Feature Feedback, Questions, & Ideas]
<p>My project site for a repo named ‘groceries’ (kosi.dev/groceries/) does not show the contents of the grocery repo. Instead, it shows the contents of the organization site (kosi.dev/).</p>
<p>I am using separate branches named ‘gh-pages’ for both repos, and the sites are written with Svelte.<br>
I read something about sirv --single catching all urls, so I removed ‘–single’ from start in package.json, with no apparent effects.</p>
<p>repos:</p><aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/kosi-dev/kosi-dev.github.io" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/46494695/181065875-b1903572-ad64-4f76-9838-48bb126b13fe.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/3/4369257981a24c442553533fbfe94fd04e8b8a7a_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/3/4369257981a24c442553533fbfe94fd04e8b8a7a_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/4/3/4369257981a24c442553533fbfe94fd04e8b8a7a.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/3/4369257981a24c442553533fbfe94fd04e8b8a7a_2_10x10.png"></div>

<h3><a href="https://github.com/kosi-dev/kosi-dev.github.io" target="_blank" rel="noopener nofollow ugc">kosi-dev/kosi-dev.github.io</a></h3>

  <p><span class="label1">Contribute to kosi-dev/kosi-dev.github.io development by creating an account on GitHub.</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/kosi-dev/groceries" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/46494695/181065876-b46a3bd2-39c9-480b-95ec-13577f6f0738.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/d/ed4e4f411fb9815b433da721c13d8dcb62dcdeaa_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/d/ed4e4f411fb9815b433da721c13d8dcb62dcdeaa_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/e/d/ed4e4f411fb9815b433da721c13d8dcb62dcdeaa.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/d/ed4e4f411fb9815b433da721c13d8dcb62dcdeaa_2_10x10.png"></div>

<h3><a href="https://github.com/kosi-dev/groceries" target="_blank" rel="noopener nofollow ugc">kosi-dev/groceries</a></h3>

  <p><span class="label1">Contribute to kosi-dev/groceries development by creating an account on GitHub.</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>How can I make the project site display it’s own content?</p>

**Comment by canuckjacq** [comment]
<p>:wave: Welcome!</p>
<p>This is hard to be certain of because of the custom domain but -</p>
<p>I’ve seen similar before and I suspect the problem is the<code>/groceries</code> folder in your organisation repository.</p>
<p>Because of the way the Pages URLs work, your organisation repository will be at</p>
<p><code>kosi-dev.github.io</code></p>
<p>And any folder in that Pages repository will show up as</p>
<p><code>kosi-dev.github.io/foldername</code></p>
<p>(in this case, your <code>index.html</code> is in the <code>public</code> folder, which then contains <code>/groceries</code>.) Your readme seems to indicate that is the default folder.</p>
<p>A separate Pages site in your organisation will be hosted at</p>
<p><code>kosi-dev.github.io/reponame</code></p>
<p>This means that you can’t have a <code>/groceries</code> folder in your organisation Pages site repository AND also have a Pages site for a repository called <code>groceries</code>.</p>
<p>Again, harder to troubleshoot with the custom domain on top of this, but changing that folder name would be the first thing I would try. Rebuid, and then clear caches and see :slight_smile:</p>

**Comment by magnetenstad** [comment, marked as answer]
<p>Thanks for the response! I was able to find a solution.</p>
<p>When building with Svelte, a <code>build</code> folder is generated. This folder is referenced to in <code>index.html</code>. The problem seems to be that the index file in the groceries repo was pointing to the build folder in the organization repo. Adding <code>groceries/</code> to the refs solved the problem.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/8/880101464d1fe13867bd0b42d923fb753c79be9e.png" data-download-href="/uploads/short-url/jp95wEdqgmRBxIF0sr8UYuAGugm.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/46494695/181065878-ab7680d8-9450-402e-9712-de7c4095ca0c.png" alt="image" data-base62-sha1="jp95wEdqgmRBxIF0sr8UYuAGugm" width="579" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/8/880101464d1fe13867bd0b42d923fb753c79be9e_2_579x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/8/880101464d1fe13867bd0b42d923fb753c79be9e_2_868x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/8/8/880101464d1fe13867bd0b42d923fb753c79be9e.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/8/880101464d1fe13867bd0b42d923fb753c79be9e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1061×915 39.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
